source/docs/software/hardware-apis/pneumatics/pneumatics.rst

```diff
@@ -99,22 +111,22 @@ Double solenoids are controlled by the DoubleSolenoid class in WPILib. These are
         // further reading is available at https://www.geeksforgeeks.org/static-import-java/
         import static edu.wpi.first.wpilibj.DoubleSolenoid.Value.*;
 
-        DoubleSolenoid exampleDouble = new DoubleSolenoid(PneumaticsModuleType.CTREPCM, 1, 2);
-        DoubleSolenoid anotherDoubleSolenoid = new DoubleSolenoid(/* The PCM CAN ID */ 9, PneumaticsModuleType.CTREPCM, 4, 5);
+        DoubleSolenoid exampleDoublePCM = new DoubleSolenoid(PneumaticsModuleType.CTREPCM, 1, 2);
+        DoubleSolenoid exampleDoublePCH = new DoubleSolenoid(/* The CAN ID */ 9, PneumaticsModuleType.REVPH, 4, 5);
```

Comment:
```suggestion
        DoubleSolenoid exampleDoublePCH = new DoubleSolenoid(PneumaticsModuleType.REVPH, 4, 5);
```

Since all of the constructors in this file optionally take a CAN ID, the `int module` parameter should be explained in a separate section at the end and shouldn't be lumped in with this.

Reply:
This is discussed above 

> Additionally, the CAN ID can be passed to the DoubleSolenoid if teams have a non-standard CAN ID.

Reply:
That holds true for both types of solenoids but is written here. The text for single and double solenoids should match in this regard.

Reply:
It's not parallel structure, but it's also covered for single solenoids

>  To construct a Solenoid object, simply pass the desired port number (assumes default CAN ID) and pneumatics module type or CAN ID, pneumatics module type, and port number to the constructor.